tests/review-child-handoff.test.sh

```
@@ -2684,13 +2700,13 @@ for doc in "$REVIEW" "$SIMPLIFY" "$POST"; do
   # the digest argument is pinned either way.
   grep -Eq 'uberdev_dispatch_child_capture "\$edge" "\$handoff" "\$handoff_sha256" "\$result" "\$(child_)?status"' "$doc"
 done
-grep -Fq "'handoff_sha256':handoff_sha256" "$REVIEW"
+grep -Fq "'handoff_sha256':handoff_sha256" "$FENCES"
 grep -Fq "'handoff_sha256':handoff_sha256" "$SIMPLIFY"
 grep -Fq 'handoff_sha256:$handoff_sha256' "$POST"
 grep -q 'REVIEW_WAIT_RC.*-ne 1' "$POST"
 [ "$(grep -c 'post_review_run_capped "' "$POST")" -eq 2 ]
 grep -q 'post_review_roster_complete "$REVIEW_LAUNCHED" "$REVIEW_EXPECTED_COUNT"' "$POST"
-! grep -En "wait_child .* 0|IFS='\\|'|additional_focus|brief_path|lens_index" "$REVIEW" "$SIMPLIFY" "$POST"
+! grep -En "wait_child .* 0|IFS='\\|'|additional_focus|brief_path|lens_index" "$REVIEW" "$FENCES" "$SIMPLIFY" "$POST"
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)